CONTRIBUTING.md

```diff
@@ -1,22 +1,53 @@
 # Contributing to Git Extensions
 
-## How to contribute code
+GitExtensions is a project with a long history, made possible by [hundreds of contributors](https://github.com/gitextensions/gitextensions/graphs/contributors).
 
-* Login in github (you need an account)
-* Fork the main repository from [github](http://github.com/gitextensions/gitextensions)
-* Read [Project workflow](https://github.com/gitextensions/gitextensions/wiki/Project-Workflow) and [Coding guide](https://github.com/gitextensions/gitextensions/wiki#coding-guide)
-* Push your changes to your fork
-* Create a pull request
+We welcome contributions including:
 
-## How to debug GitExtensions
+- Ideas for improvements
+- Bug reports
+- Bug fixes
+- New features
 
-The installer is build using WiX. You need to install WiX when you want to build the installer. This can be downloaded here: [http://wixtoolset.org/](http://wixtoolset.org/). If you do not want to build the installer, just open the solution and ignore the warning.
+To help the project maintainers be as effective as possible, please follow some simple guidelines.
 
-* Open the solution file (GitExtensions.sln)
-* Hit F5 to compile and run GitExtensions
+## Reporting Issues
 
-## How to create the installer
+[Search the issue tracker](https://github.com/gitextensions/gitextensions/issues?&q=) for an
+existing or closely related issue before creating a new one. Be sure to include closed issues
+in your search.
 
-* Download and install WiX [http://wixtoolset.org/](http://wixtoolset.org/)
-* Run Setup\\BuildInstallers.cmd to build the installers
+If an issue already exists, read through the discussion. If you can something helpful, do so.
+You can vote on issues that you'd like to see prioritised.
 
+If no issue exists, create one. Complete the template, and add any further information that
+could be relevant.
+
```

Comment:
I would have liked to have a suggestion to try the latest portable build too.

Reply:
I am on a fence. I think this is a good idea, but there is also a downside...

* ❓ There are two portable builds - v2.5x and v3.x - which one do we link?
* 👎 Canary builds may be unstable (the reason this got voted down), however
* 👍 Canary builds may contain fixes to users issues

Reply:
Suggest a change to the following:

You may also test the portable builds (no installer) from the latest merged changes: [Windows](https://ci.appveyor.com/project/gitextensions/gitextensions/branch/master/artifacts) and [Mono](https://ci.appveyor.com/project/gitextensions/gitextensions/branch/release/2.51/artifacts).

Reply:
Good idea. I've added another paragraph explaining that if you find a closed issue then you might like to try a CI build, that links to a [new wiki page](https://github.com/gitextensions/gitextensions/wiki/CI-Builds). We can then add as much info as needed on that page without cluttering this document. You all have a better idea of what content should be there, so please edit as you see fit.

Reply:
I find references to Windows and Mono confusing... The idea is good, we need to make it clearer

Reply:
Suggestion:
You may also test the portable builds (no installer) from the latest merged changes: (drews wiki page, I updated it slightly)